src/bootstrap/config.toml.example

```diff
@@ -14,6 +14,9 @@
 # =============================================================================
 [llvm]
 
+# Indicates whether rustc will support compilation with LLVM (currently broken)
```

Comment:
The comment seems to imply that the build *with* llvm is broken, not the build without it. Should be the other way round, right?